test/wizard.jl

```diff
@@ -252,5 +252,8 @@ end
 @testset "GitHub - authentication" begin
     withenv("GITHUB_TOKEN" => "") do
         @test BinaryBuilder.github_auth(allow_anonymous=true) isa GitHub.AnonymousAuth
+        input_stream = IOBuffer()
+        close(input_stream)
+        @test_throws ErrorException BinaryBuilder.obtain_token(ins=input_stream)
```

Comment:
Can you also test a successful invocation here?

Reply:
I'm not sure this is easy: `_getpass` requires a `Base.TTY` as input stream, and even if I knew how to create a custom `Base.TTY` and how to write there, I'd need real GitHub credentials not to get an error :disappointed: